eng/native/configurecompiler.cmake

```diff
@@ -1027,3 +1034,8 @@ elseif (NOT CLR_CMAKE_HOST_BROWSER AND NOT CLR_CMAKE_HOST_WASI)
     enable_language(ASM)
 
 endif(CLR_CMAKE_HOST_WIN32)
+
+if (INTERNAL_ZLIB_NG)
```

Comment:
This symbol says whether to use zlib-ng from src\native\external directory. There is a mild confusion between internal vs. external.

Other similar symbols have opposite polarity: `CLR_CMAKE_USE_SYSTEM_BROTLI`, `CLR_CMAKE_USE_SYSTEM_LIBUNWIND`. Should we match the pattern here?

Reply:
Also, the matching managed symbol is called `UseSystemZlib`. I think this should be called `CLR_CMAKE_USE_SYSTEM_ZLIB` to make it easy to connect the two together.

Reply:
> CLR_CMAKE_USE_SYSTEM_ZLIB 

That's a good suggestion. Applying it now.

Reply:
And I see what you mean about the confusion, I accidentally used them interchangeably which is wrong. I need to fix that.

Reply:
Resolved. Would you mind taking another look, @jkotas?